docgen/src/Styling.md

```diff
@@ -10,58 +10,133 @@ All widgets that render one or more DOM nodes accept a `theme` prop. This prop i
 The different theme keys supported by components are described on their respective documentation page. You can also provide themes using css-modules, Aphrodite, JSS and others if you want. 
```

Comment:
It feels odd to me that a tutorial on theming would start with that. What about something more general and more indicative that it is a tutorial? For example:

```markdown
#Welcome to the react-instantsearch theming tutorial

react-instant aims to provide the best to make the best search experience your way! Theming 
is a big part of customising the search experience. That's why react-instantsearch uses
[themeable](link to themeable website) to provides easily customizable widgets. In this tutorial,
we'll cover only css modules theming options with themeable, but it supports much more
options that might fit your current stack better (see their [website]())
```